kubectl-rook-ceph.sh

```diff
@@ -273,6 +276,97 @@ function run_purge_osd() {
     rook ceph osd remove --osd-ids=$1 --force-osd-removal=$force_removal"
 }
 
+####################################################################################################
+# 'kubectl rook-ceph debug commands
+####################################################################################################
+
+function update_deployment_spec_command(){
+  deployment_spec=$1
+  main_command='["sleep","infinity"]'
+  deployment_spec=$(jq .template.spec.containers[0].command="$main_command" <<< $deployment_spec)
+  deployment_spec=$(jq .template.spec.containers[0].args=[] <<< $deployment_spec)
+  echo "$deployment_spec" # for returning the updated deployment_spec
+}  
+
+function update_deployment_spec_image(){
+  deployment_spec=$1
+  alternalte_image=\"$2\"
+  deployment_spec=$(jq .template.spec.containers[0].image="$alternalte_image" <<< $deployment_spec)
+  echo "$deployment_spec" # for returning the updated deployment_spec
+}
+
+function verify_deployment_exist(){
+  if [[ $deployment_name != *"mon"* ]] && [[ $deployment_name != *"osd"* ]]; then
+    fail_error "only mon or osd deployment name can passed for starting debug mode"
+  fi
+  if _="$(KUBECTL_NS_CLUSTER get deployment "$deployment_name")"; then
```

Comment:
If the intent is just to swallow the console output, it is likely better to use `KUBECTL_NS_CLUSTER get deployment "$deployment_name" &>/dev/null`

```suggestion
  if KUBECTL_NS_CLUSTER get deployment "$deployment_name" &>/dev/null; then
```

Reply:
DOne, 
Anyway if I can print only false condition here,

Currently I need to go for if and else and print both it is present and not present